dev-utils/saucelabs.js

```diff
@@ -1,60 +0,0 @@
-/**
- * MIT License
- *
- * Copyright (c) 2017-present, Elasticsearch BV
- *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- *
- */
-
-function launchSauceConnect(userConfig, done) {
-  var sauceConnectLauncher = require('sauce-connect-launcher')
-
-  var config = {
-    username: userConfig && (userConfig.username || process.env.SAUCE_USERNAME),
-    accessKey:
-      userConfig && (userConfig.accessKey || process.env.SAUCE_ACCESS_KEY),
-    logger: console.log,
-    noSslBumpDomains: 'all'
-  }
-
-  var tryConnect = function(maxAttempts, currAttempts, done) {
```

Comment:
We should use this script to `launchSauceConnect` the logic here make sure to attempt three times if the first try fails!

Reply:
Retry strategy is already handled here https://github.com/elastic/apm-agent-js-base/blob/982846b9d351ea5fd99800295da14a9e7e581f49/dev-utils/test-config.js#L34